services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/DownloadDirectoryRequest.java

```diff
@@ -204,86 +173,6 @@ public interface Builder extends CopyableBuilder<Builder, DownloadDirectoryReque
          */
         Builder bucket(String bucket);
 
-        /**
-         * Specify the key prefix for the virtual directory. If not provided, all subdirectories will be downloaded recursively
-         * <p>
-         * See <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-prefixes.html">Organizing objects using
-         * prefixes</a>
-         *
-         * <p>
-         * When a non-empty prefix is provided, the prefix is stripped from the directory structure of the files.
-         * <p>
-         * For example, assume that you have the following keys in your bucket:
-         * <ul>
-         *     <li>sample.jpg</li>
-         *     <li>photos/2022/January/sample.jpg</li>
-         *     <li>photos/2022/February/sample1.jpg</li>
-         *     <li>photos/2022/February/sample2.jpg</li>
-         *     <li>photos/2022/February/sample3.jpg</li>
-         * </ul>
-         *
-         * Give a request to download the bucket to a destination with a prefix of "/photos" and destination path of "test", the
-         * downloaded directory would like this
-         *
-         * <pre>
-         *   {@code
-         *      |- test
-         *         |- 2022
-         *              |- January
-         *                 |- sample.jpg
-         *              |- February
-         *                 |- sample1.jpg
-         *                 |- sample2.jpg
-         *                 |- sample3.jpg
-         *   }
-         * </pre>
-         * @param prefix the key prefix
-         * @return This builder for method chaining.
-         */
-        Builder prefix(String prefix);
-
-        /**
-         * Specify the delimiter that will be used to retrieve the objects within the provided bucket. A delimiter causes a list
-         * operation to roll up all the keys that share a common prefix into a single summary list result. It's null by default.
-         *
-         * For example, assume that you have the following keys in your bucket:
-         *
-         * <ul>
-         *     <li>sample.jpg</li>
-         *     <li>photos-2022-January-sample.jpg</li>
-         *     <li>photos-2022-February-sample1.jpg</li>
-         *     <li>photos-2022-February-sample2.jpg</li>
-         *     <li>photos-2022-February-sample3.jpg</li>
-         * </ul>
-         *
-         * Give a request to download the bucket to a destination with delimiter of "-", the downloaded directory would look
```

Comment:
Can we move these (well-written 😛 ) Javadocs to ListObjectV2Transformer?

Reply:
Done